specreduce/utils/__init__.py

```diff
@@ -1,3 +1,5 @@
 """
 General purpose utilities for specreduce
 """
+
+from .utils import *  # noqa
```

Comment:
do we want relative imports here?

Reply:
i'm not sure?

Reply:
I don't have an opinion either... we can always change this later if we come up with a policy/standard we want to follow, right?

Reply:
well, the PEP8 guide recommends absolute imports, fwiw:  https://peps.python.org/pep-0008/#imports.